cmake/CodeStyle.cmake

```diff
@@ -19,23 +19,52 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-option(APPLY_CPP_CODE_STYLE "Apply C++ code style using clang format" OFF)
+option(APPLY_CPP_CODE_STYLE "Apply C++ code style using clang format" ON)
+option(CHECK_CPP_CODE_STYLE "Check C++ code style using clang format" ON)
```

Comment:
Does this make sense to default to `ON`?  If I'm reading the below correctly, if `APPLY_CPP_CODE_STYLE` is `ON`, `CHECK_CPP_CODE_STYLE` will be forced to `OFF`.

Reply:
hi, it should be off, I have changed the default to off. thanks

Reply:
Can the code style check be disabled by default in the CMake configuration files, and then enabled by default by the build script?  The intent would be for the default CMake output to perform a  minimum number of build steps, with a minimum number of tool dependencies, for the end user that just wants to check out and build the code.  The build script would then be used by developers contributing to the project to verify changes before they are submitted.

Reply:
sure. it is already enabled by default in the build script, and I just made a commit 
[cedbfcb](https://github.com/LunarG/gfxreconstruct/pull/258/commits/cedbfcba02d9e340e0d7c1a8e4d128497ab353cc) to disable it by default in the CMake.